openid.js

```diff
@@ -47,20 +48,21 @@ module.exports = function (RED) {
     const redirect_uri = req.query.callback
     const client_id = req.query.clientId
     const client_secret = req.query.clientSecret
+    const scopes = req.query.scopes
```

Comment:
As you have put a placeholder in the HTML form, can you set the default value? 

Example:

```suggestion
    const scopes = req.query.scopes.trim() !== '' ? req.query.scopes.trim() : 'openid email offline_access'
```